mdm_profiles/setup_assistant.json

```diff
@@ -0,0 +1,28 @@
+{
+	"profile_name": "FleetDM example enrollment profile",
+	"allow_pairing": true,
+	"is_mdm_removable": true,
+	"org_magic": "1",
+	"language": "en",
+	"region": "US",
+	"skip_setup_items": [
+		"Accessibility",
+		"Appearance",
+		"AppleID",
+		"AppStore",
+		"Biometric",
+		"Diagnostics",
+		"FileVault",
+		"iCloudDiagnostics",
+		"iCloudStorage",
+		"Location",
+		"Payment",
+		"Privacy",
+		"Restore",
+		"ScreenTime",
+		"Siri",
+		"TermsOfAddress",
+		"TOS",
+		"UnlockWithWatch"
+	]
+}
```

Comment:
@noahtalerman I added this json as setup assistant example, let me know if that's what you had in mind (it's the default profile that Fleet uses).

Reply:
Nice! That looks good to me.

@lukeheath just checking, will putting this example in `/mdm_profiles` mess w/ the GitHub action that applies config profiles?

Reply:
Yeah the workflow only considers `.moblieconfig`, this should be fine.